### PR TITLE
Address the rest of the themes to correct Overlay awareness...

### PR DESCRIPTION
**Pull request by Syres916**
...and generally make them ready for 1.0 Release with regard to new Start page, default material appearance, spreadsheet text colors and Techdraw background.

Also noticed that the _check_light.svg_ icon was actually dark so corrected that at the same time.

### CODE REVIEW

**Comment by Syres916** [comment]
@kadet1090 if you get chance over the next few days, could you please review, TIA.

File: Dark-contrast/Dark-contrast.qss [hunk 1]
```diff
@@ -94,7 +94,7 @@ QToolBar  {
 }
 
 QMdiArea {
-    background-image: url(qss:images_dark-light/background_freecad_dark.svg);
+    background-image: url(qss:/../images_dark-light/background_freecad_dark.svg);
```
Comment (kadet1090) [inline Dark-contrast/Dark-contrast.qss:97]: I suppose that you tested it, and it works but this syntax seems really strange - this is one directory above the top one which is not a thing, why is it done like that?

**Review by kadet1090 — APPROVED** [review]

**Comment by kadet1090** [comment]
Done, I was on my vacation 2 weeks ago and must have missed that e-mail. I need to learn to take more care of built-in github notifications.

File: Dark-contrast/Dark-contrast.qss (hunk 1 above)
Comment (Syres916) [reply Dark-contrast/Dark-contrast.qss]: Functionality it's no difference, it was my Dos 3.1 heritage coming to the surface, the shortened path just doesn't look correct to my vintage, all back to shortened paths now.

**Pull request merged**